src/game-logic/cows/cow-selection.test.js

```diff
@@ -0,0 +1,113 @@
+import { act, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+
+import { farmhandStub } from '../../test-utils/stubs/farmhandStub'
+import { generateCow, getCowDisplayName } from '../../utils'
+import { stageFocusType } from '../../enums'
+
+describe('cow selection', () => {
+  const cowId1 = 'foo'
+  const cowId2 = 'bar'
+  const cowStub1 = generateCow({ id: cowId1 })
+  const cowStub2 = generateCow({ id: cowId2 })
```

Comment:
can this be moved into the `beforeEach`? the tests in this file look safe enough but i find it's best to have things like this reset between each spec to avoid accidentally changing something in one test that sticks to the next